package.json

```diff
@@ -16,6 +16,8 @@
     "test": "hardhat test",
     "test-e2e": "mocha --config .mocharc-e2e.js",
     "test:fix": "yarn lint:fix && yarn format:fix && yarn test && yarn typecheck",
+    "solc": "pip3 install solc-select && solc-select install 0.7.6",
```

Comment:
Not sure if this makes sense to add here. Probably want to have this install step under the README setup instructions.

Reply:
I will add the instructions in the README

Reply:
Done.